packages/@aws-cdk/aws-eks/README.md

```diff
@@ -196,7 +196,7 @@ const lt = new ec2.CfnLaunchTemplate(this, 'LaunchTemplate', {
   },
 });
 this.cluster.addNodegroup('extra-ng', {
-  launchTemplate: {
+  launchTemplateSpec: {
```

Comment:
Decided to go with instead of hoisting the properties up so as to not make both `launchTemplateId` and `launchTemplateVersion` optional.

We will deprecate this in favor of a `launchTemplate` property that will accept an `ec2.ILaunchTemplate` when it's implemented.